packages/edge-bundler/package.json

```diff
@@ -42,17 +42,14 @@
     "test": "test/node"
   },
   "devDependencies": {
-    "@types/glob-to-regexp": "^0.4.1",
     "@types/node": "^14.18.32",
     "@types/semver": "^7.3.9",
     "@types/uuid": "^10.0.0",
     "@vitest/coverage-v8": "^0.34.0",
     "archiver": "^5.3.1",
     "chalk": "^5.4.0",
-    "cpy": "^9.0.1",
-    "cross-env": "^7.0.3",
     "nock": "^13.2.4",
-    "tar": "^6.1.11",
+    "tar": "^7.4.3",
```

Comment:
I had erroneously removed this and it got upgraded when I readded it. It looks fine so I'll keep the upgrade: https://github.com/isaacs/node-tar/blob/main/CHANGELOG.md#70.

Reply:
😓 never mind: https://github.com/isaacs/node-tar/pull/435